arxivdigest/frontend/templates/base.html

```diff
@@ -91,7 +88,15 @@
 
 <footer>
     <div class="container ">
-        <a href="{{ url_for('general.feedbackPage') }}" id="footer_feedback">Give us feedback on our system.</a>
+        <a href="{{ url_for('general.feedbackPage') }}" class="footer_item">Give us feedback on our system</a>
+        <p class="footer_item">|</p>
+        <a href="{{ url_for('general.termsandconditions') }}" class="footer_item">Terms and conditions</a>
+        <p class="footer_item">|</p>
+        <a href="{{ url_for('general.registerSystemPage') }}" class="footer_item">Living Labs</a>
+        <p class="footer_item">|</p>
+        <a href="https://www.uis.no/frontpage/#Studies" class="footer_item">uis.no</a>
+        <p class="footer_item">|</p>
+        <p class="footer_item">Copyright (c) IAI group, University of Stavanger, 2020</p>
```

Comment:
<a href="http://iai.group/">IAI group</a>, <a href="https://www.uis.no/">University of Stavanger</a>, 2020